public/class-media-search-enhanced.php

```diff
@@ -161,9 +161,15 @@ public static function posts_clauses( $pieces ) {
 			}
 
 			if ( ! empty( $vars['post_mime_type'] ) ) {
-				// Use esc_like to escape slash
-				$like = '%' . $wpdb->esc_like( $vars['post_mime_type'] ) . '%';
-				$pieces['where'] .= $wpdb->prepare( " AND $wpdb->posts.post_mime_type LIKE %s", $like );
+					$mime_types = is_array( $vars['post_mime_type'] ) ? $vars['post_mime_type'] : [ $vars['post_mime_type'] ];
+					$like_clauses = array();
+
+					foreach( $mime_types as $mime_type ) {
+							$mime_type_like = '%' . $wpdb->esc_like( $mime_type ) . '%';
+							$like_clauses[] = "$wpdb->posts.post_mime_type LIKE '$mime_type_like'";
+					}
+
+					$pieces['where'] .= " AND (" . implode( ' OR ', $like_clauses ) . ")";
```

Comment:
Are the indents here a bit off?

Also, mind if changing `$like_clauses[] = "$wpdb->posts.post_mime_type LIKE '$mime_type_like'";` to use `$wpdb->prepare()`?

Reply:
Hi @yoren 
prepare has been added and indentation fixed.
Would be great if we had a `.editorconfig` or prettier file to keep the pattern `.prettierrc`

Reply:
Thank you. I'll do some test and merge if no further issues + do the release to wp.org over the weekend.